mix.exs

```diff
@@ -47,7 +47,7 @@ defmodule Opencensus.Absinthe.MixProject do
 
   defp deps() do
     [
-      {:absinthe_plug, "~> 1.4.0", only: :dev, runtime: false},
+      {:absinthe_plug, "~> 1.4.0", only: [:dev, :test], runtime: false},
```

Comment:
My tests are that comprehensive? I only just got to writing the Phoenix integration test in `opencensus_honeycomb`. :/

Reply:
I will check again but pretty sure I had to do this to make the tests pass.

Reply:
It was actually to remove a warning:

```
warning: function Absinthe.Plug.default_pipeline/2 is undefined (module Absinthe.Plug is not available)
  lib/opencensus/absinthe/plug.ex:27
```

Reply:
You might want `optional: true` instead of `only: [:dev, :test]`.